react-js/react-part-1-intro.md

```diff
@@ -38,85 +37,39 @@ React allows developers to create large web applications that can change data, w
 - What is JSX and why is it different than HTML?
 - Specifics about co-locating markup with logic in code.
 
-### Lesson Materials
+## Lesson
 
-You will spend the day working through these materials. 
+### Guided Practice
 
-1. [Introduction to React (video)](https://youtu.be/ycstRj2i66k)
-1. [Learn React in 5 minutes (article)](https://medium.freecodecamp.org/learn-react-js-in-5-minutes-526472d292f4)
-1. [Techtonica React slides](https://docs.google.com/presentation/d/1Bswkl7e1kGVav7KFabHrBgXhRi7mlINTbinxnUcGDy8/edit?usp=sharing)
-1. [Building your first component in React.js | Pluralsight (video)](https://youtu.be/K_jS1anlVAM)
-1. [Work through this React tutorial](https://facebook.github.io/react/index.html)
-1. [CodePen for Hello World Example](http://codepen.io/marcacyr/pen/NAyqgX)
-1. [CodePen Greeting with Input Example](http://codepen.io/marcacyr/pen/bZLVbj)
-1. [CodePen for Counter Example](http://codepen.io/marcacyr/pen/rLJVqR)
-1. [CodePen for List - Rendering using a loop in React](http://codepen.io/marcacyr/pen/KrQpYb)
+1. Watch [Introduction to React (12 min video)](https://youtu.be/ycstRj2i66k)
+    - Note that "MVC" stands for Model, View, Control.
+1. Spend just 5 minutes on this CodePen example of multiple instances of the same React component: a blue box. Press the "fork" button at the bottom to make a copy for yourself, and then change it as much as you like.  https://codepen.io/alodahl/pen/xxgoeqL
```

Comment:
I feel like whenever I tell participants to "play around" they get really confused. Should we clarify what they should do more.  like:  add a fourth box, make a new greenBox component that show a green outline, etc?